test/props_evm.test.js

```diff
@@ -39,137 +39,139 @@ const { isAwaitExpression } = require('typescript');
         assert.equal(constractTreasury, 0x0);
         await props.connect(deployer).changeTreasury(treasury_address);
       });
-      describe('Basic Functionality Test', function () {
-        it('Initializes the PROPS Correctly.', async () => {
+      describe("Basic Functionality Test", function () {
+        it("Initializes the PROPS Correctly.", async () => {
           const current_supply = await props.current_supply();
-          assert.equal(current_supply, ethers.toBigInt('0'));
+          assert.equal(current_supply, ethers.toBigInt("0"));
           const mint_tranche_limit = await props.mint_tranche_limit();
-          assert.equal(mint_tranche_limit, ethers.toBigInt('200000000000000'));
+          assert.equal(mint_tranche_limit, ethers.toBigInt("200000000000000"));
           const last_mint_timestamp = await props.last_mint_timestamp();
-          assert.equal(last_mint_timestamp, ethers.toBigInt('0'));
+          assert.equal(last_mint_timestamp, ethers.toBigInt("0"));
           const last_mint_tranche_timestamp =
             await props.last_mint_tranche_timestamp();
-          assert.equal(last_mint_tranche_timestamp, ethers.toBigInt('0'));
+          assert.equal(last_mint_tranche_timestamp, ethers.toBigInt("0"));
           const admin = await props.admin();
           assert.equal(admin, deployer.address);
           const minter = await props.minter();
           assert.equal(minter, 0x0);
           const limiter = await props.limiter();
           assert.equal(limiter, 0x0);
 
-          const admin_addr = await props.getRoleAdmin(
-            ethers.keccak256(ethers.toUtf8Bytes('ADMIN_ROLE'))
-          );
-          assert(admin_addr, deployer.address);
+          const is_admin = await props.hasRole(ethers.keccak256(ethers.toUtf8Bytes("ADMIN_ROLE")), deployer.address);
+          assert.equal(is_admin, true);
 
           const decimals = await props.decimals();
           assert(decimals, 8);
+
+          const DEFAULT_ADMIN_ROLE = '0x' + '00'.repeat(32);
```

Comment:
@harispericorn  This assertion should fail as its a 0 address, right?

Reply:
bytes32 public constant DEFAULT_ADMIN_ROLE = 0x00;
defualt admin role has bytes zero address